stablehlo/dialect/Version.cpp

```diff
@@ -83,9 +83,9 @@ Version Version::fromCompatibilityRequirement(
     case CompatibilityRequirement::NONE:
       return Version::getCurrentVersion();
     case CompatibilityRequirement::WEEK_4:
-      return Version(1, 7, 5);  // WEEK_4 ANCHOR: DO NOT MODIFY
+      return Version(1, 7, 7);  // WEEK_4 ANCHOR: DO NOT MODIFY
```

Comment:
NIT: comment "DO NOT MODIFY" can be removed?  we are, will be modifying the versions.

Reply:
Sorry for confusion
`ANCHOR: DO NOT MODIFY` - it means do not modify the comment (not the value)

`ANCHOR: DO NOT MODIFY` is used by script which will automatically bump the versions in future - https://github.com/openxla/stablehlo/blob/main/build_tools/update_version_h_cpp.sh